application/src/js/tablebuilder2/tablebuilder2.js

```diff
@@ -61,14 +61,13 @@ function getIsSimpleData(data) {
     return values.length === data.length - 1
 }
 
-$(document).ready(function () {
+document.addEventListener('DOMContentLoaded', function() {
 
     // add events to buttons
-    $('#preview').click(preview);
-    $('#confirm-data').click(setTableData);
-    $('#edit-data').click(editTableData);
-    $('#cancel-edit-data').click(cancelEditData);
-    $('#save').click(saveTable);
+    document.getElementById('confirm-data').addEventListener('click', setTableData)
```

Comment:
Missed out the `#preview` click here?

Reply:
@samuelhwilliams that was deliberate - in that when I changed it to `document.getElementById('preview')` it revealed that there _is_ no preview link any more...